### PR TITLE
remove  #StandWithUkraine banner to updates

### PR DESCRIPTION
**Pull request by michealzh**
The OSS is no need the WAR!!

### CODE REVIEW

**Review by podarok — CHANGES_REQUESTED** [review]
Let's avoid bad behavior for hiding a war activity in OSS. 
Users must know how bad is russia and why support of Ukraine is so important

**Comment by Max-Kuzomko** [comment]
War in Ukraine - is a war against the whole world. Ukrainians were viciously attacked at night, like hitler did it. Ukrainians defend freedom, freedom of choice, democracy for the whole world. They don't want a soviet union come back. It's not a secret if Ukraine fall - putin will attack Europe next. russia and belarussia has already started a hybrid war against the world. You can see tons of fake news and so on.

It's not a time to demonstrate neutral status. It's a time for support. As phrases like "I'm out of politics", "I am neutral" lead to a war. It's not acceptable to say I am neutral when russian troops are killing kids, civilians, bombing ukrainian cities.

**Comment by podarok** [comment]
BTW, I highly doubt you, @michealzh , are a good fit for Open Source Software.
Open - means open. Open access to the information.
Your activity is against Open Source principles. You are trying to close and hide.

I vote to ban you from GitHub

File: src/Package/SymlinkDumper.php [hunk 1]
```diff
@@ -318,7 +318,6 @@ public function dump(array $packageIds, bool $force = false, bool $verbose = fal
             $this->rootFile['providers-api'] = str_replace('VND/PKG', '%package%', $this->router->generate('view_providers', ['name' => 'VND/PKG', '_format' => 'json'], UrlGeneratorInterface::ABSOLUTE_URL));
             $this->rootFile['warning'] = 'Support for Composer 1 is deprecated and some packages will not be available. You should upgrade to Composer 2. See https://blog.packagist.com/deprecating-composer-1-support/';
             $this->rootFile['warning-versions'] = '<1.99';
-            $this->rootFile['info'] = json_decode('"\u001b[37;44m#StandWith\u001b[30;43mUkraine\u001b[0m"');
```
Comment (lialiole) [inline src/Package/SymlinkDumper.php:321]: Stop Chinese terrorists. Read about  uyghur genocide

**Review by lialiole — CHANGES_REQUESTED** [review]
Re

**Comment by johnSmithA2** [comment]
> 

Did you stand with the Afghans? Did you stand with the Iraqis? Did you stand with the Palestinians? I don't mean to offend Ukrainians, but why do they only stand with Ukrainians? If the United States is fighting with Ukraine now, how many people will stand with you? Hypocritical people, ridiculous double standards.

**Comment by johnSmithA2** [comment]
> BTW, I highly doubt you, @michealzh , are a good fit for Open Source Software. Open - means open. Open access to the information. Your activity is against Open Source principles. You are trying to close and hide.
> 
> I vote to ban you from GitHub

@podarok Did you stand with the Afghans? Did you stand with the Iraqis? Did you stand with the Palestinians? I don't mean to offend Ukrainians, but why do they only stand with Ukrainians? If the United States is fighting with Ukraine now, how many people will stand with you? Hypocritical people, ridiculous double standards.

**Comment by JiaMorant** [comment]
To all who stand with Ukraine. Will you stand with Afghans? Are you anti American? Will you stand with the Palestinians? Are you anti Israel? If your answers are all "yes", you are a respectable person.

**Review by 18o — APPROVED** [review]

**Comment by dixyes** [comment]
> BTW, I highly doubt you, @michealzh , are a good fit for Open Source Software. Open - means open. Open access to the information. Your activity is against Open Source principles. You are trying to close and hide.
> 
> I vote to ban you from GitHub

So should you represent others‘ opinion in OSS which is "Open access to the information" for everyone?

How Russian programmers will think if you "StandWithUkraine"?

How Ukraine programmers will think if you "StandWithRussian"?

Let's say there's a new war between china an us:

How Chiness programmers will thing if you "StandWithUS"?

How US programmers will thing if you "StandWithChina"?

Make OSS, not WAR.

If you really really what to show your option, please not represent anyone else.

File: src/Package/SymlinkDumper.php (hunk 1 above)
Comment (dixyes) [reply src/Package/SymlinkDumper.php:321]: Yes, all Chinese is evil!

Chinese do slave trade, and force Africans work at their farm, they murdered Martin Luther King Jr. in China. Their police murdered George Floyd!

**Comment by zoujingli** [comment]
I think we just have different positions. Open source should not be of a political nature. War will always be the suffering of ordinary people. The information we receive is always what the upper level wants us to know. There are many kinds of theories about the origin of this war. Some countries are united with each other, and even have forces who want to break the situation of world peace. For example: restrict other countries' scientific and technological discoveries, carry out hegemonic acts such as economic control, and develop their own military forces around other countries. All this will inevitably make some countries become cannon fodder and ordinary people suffer.

**Comment by DeVoresyah** [comment]
> BTW, I highly doubt you, @michealzh , are a good fit for Open Source Software. Open - means open. Open access to the information. Your activity is against Open Source principles. You are trying to close and hide.
> 
> I vote to ban you from GitHub

lmao, why do people never do #StandWithPalestine ? more people got killed in Palestine because of Israel

**Comment by DeVoresyah** [comment]
> @michealzh
> 
> I had to relocate my family(3months baby girl, 4.5yo twins, and a wife) from the place where I lived in my house to the west of Ukraine.
> 
> I'm OSS developer whole my life, you can check my profile here as well as at https://www.drupal.org/u/podarok
> 
> And I must say - you are making huge mistake by keeping yourself neutral and by trying to hide a war.
> 
> OSS must be the frontline of support any movement against aggression

have u ever do the same thing for Palestine? #StandWithPalestine ? many people there got killed, or u just silent?

**Comment by Seldaek** [comment]
Quoting someone else who said it well, I won't link tho because I don't want to send the haters his way..

```
  ;; You might be reading this because you think it is not appropriate
  ;; to mix politics with software development.
  ;; Or that I should have added banners like #StandWithPalestine or #StandWithYemen,
  ;; and adding it now makes me a hypocrite. And you are right.
  ;; I understand the reasoning, but I decided to add the banner now, because this
  ;; time people I hold very dear are in Kyiv, fighting for their family, their
  ;; lives and their country. Because I know them well, I am
  ;; personally affected.
  ;; So if you disagree, just deal with it, or stop using this tool.
```

It's a mix of that, and a feeling that this war could have a much bigger/more global impact than others. Opinions my own, feel free to disagree.

**Pull request closed without merging**